pillar_queue/lib.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from pillar_queue import Queue
+
+
+def create_queue(name, aws_access_key=None, aws_access_secret=None):
+    '''
+    Shortcut function to create a queue
+    '''
+
+    if aws_access_key is None:
+        aws_access_key = os.environ.get('AWS_ACCESS_KEY_ID')
+
+    if aws_access_secret is None:
+        aws_access_secret = os.environ.get('AWS_SECRET_ACCESS_KEY')
+
+    q = Queue(
+        name=name,
+        aws_access_key=aws_access_key,
+        aws_access_secret=aws_access_secret
+    )
+
+    return q
```

Comment:
When the q is returned is it guaranteed to exist on aws? If not I think we should maybe wait to return until it is available.

Reply:
If the queue does not exist, an error is thrown.

Reply:
I think that this is fine, though maybe we should mention in the documentation that it is assumed to exist on AWS. For that reason I am changing the name to `get_queue` rather than `create_queue`, as the latter may create confusion.